lib/docker/test_env/job-distributor/go.mod

```diff
@@ -0,0 +1,321 @@
+module github.com/smartcontractkit/chainlink-testing-framework/job-distributor
```

Comment:
is this right or should I change it to github.com/smartcontractkit/chainlink-testing-framework/lib/job-distributor?